AuthenticationOverview/ServiceAccountIntegration.md

```diff
@@ -1,109 +1,102 @@
 
-`cd C:/libs/openssl-1.1.1-win64-mingw/`
+    b. Extract the folder and copy it to the **C:/libs/** location.
 
-```
-openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout private.key -out certificate_pub.crt
-```
+    c. Open a command-line window and execute the following commands:  
+      
+      `set OPENSSL_CONF=C:/libs/openssl-1.1.1-win64-mingw/openssl.cnf`  
+        
+    `cd C:/libs/openssl-1.1.1-win64-mingw/`  
+    
+    `openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout private.key -out certificate_pub.crt`
 
-<kbd>![Generate public certificate windows](../Images/auth_jwtqs_000.png "Generate Public certificate windows")</kbd>
+    <kbd>![Generate public certificate windows](../Images/auth_jwtqs_000.png "Generate Public certificate windows")</kbd>
 
-- Once you&rsquo;ve completed the steps for your chosen platform, continue in the Adobe I/O Console.
+3. Once you&rsquo;ve completed the steps for your chosen platform, continue in the Adobe I/O Console.
 
-### Step 3: Configure an API Key Integration
+### Step 3: Configure an API Key integration
 
-- Upload the public certificate (certificate_pub.crt) as a part of creating the integration.
+1. Upload the public certificate (certificate_pub.crt) as a part of creating the integration.
 
     <kbd>![Upload public certificate](../Images/auth_jwtqs_03.png "Upload public certificate")</kbd>
 
-- Your integration should now be created with the appropriate public certificate and claims.
+2. Your integration should now be created with the appropriate public certificate and claims.
 
     <kbd>![Integration created](../Images/auth_jwtqs_04.png "Integration created")</kbd>
     
 ### Step 4: Try It
 
-- Go to the JWT tab and paste in you private key to generate a JWT.
+1. Go to the JWT tab and paste in you private key to generate a JWT.
 
     <kbd>![JWT tab](../Images/auth_jwtqs_05.png "JWT tab")</kbd>
 
-- Copy the &ldquo;Sample CURL Command&rdquo; to get your first access token. 
+2. Copy the &ldquo;Sample CURL Command&rdquo; to get your first access token. 
 
     <kbd>![Get access token](../Images/auth_jwtqs_06.png "Get access token")</kbd>
 
 
-- Open Postman, then click Import &gt; Paste Raw Text and paste the copied curl command.
+3. Open Postman, then click Import &gt; Paste Raw Text and paste the copied curl command.
 
     <kbd>![Postman import](../Images/auth_jwtqs_07.png "Postman import")</kbd>
 
-- Click Send.
+4. Click Send.
 
     <kbd>![Postman send](../Images/auth_jwtqs_08.png "Postman send")</kbd>
 
-The example curl sends a POST request to [https://ims-na1.adobelogin.com/ims/exchange/jwt](https://ims-na1.adobelogin.com/ims/exchange/jwt) with the following parameters.
+    The example curl sends a POST request to [https://ims-na1.adobelogin.com/ims/exchange/jwt](https://ims-na1.adobelogin.com/ims/exchange/jwt) with the following parameters.
 
 | Parameter | Description|
```

Comment:
@MattLusher I think this section should be after the image in bullet point 3

Reply:
Not sure I agree. The statement describes what the curl is doing and leads directly into the table following. If we moved it, we'd need to move the table as well, and that would interrupt the step sequence.

Reply:
It looks confusing to me. Step 3 is the curl statement that uses those parameters. The table is after step 4 which is the results of the curl statement. When you look at the image after step 4 it shows `token_type`, `access_token` and `expires_in` not `client_id`, `client_secret` and `jwt_token`. I'd also move the table above step 4, that wasn't clear from my earlier comment.

Reply:
I agree with @macdonst, the table at the end should be part of step 3 as per the logical sequence.

Reply:
I bow to the majority. ;-) I've just added a commit to make that change.

Reply:
@MattLusher Thank you!

Reply:
@macdonst @hirenoble  Thanks, guys! Now, per the process Kanika laid out, if we're ready to publish, rather than PR from stage to master (which would bring over the stage version of the manifest.json and screw up the live site), we PR from dev to master. I'll set up that PR now.